lib/Doctrine/Common/Persistence/Mapping/Driver/StaticPHPDriver.php

```diff
@@ -62,6 +65,7 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
 
     /**
      * {@inheritDoc}
```

Comment:
Wasn’t there something about inheritdoc being wrong in general?

Reply:
Yes, but we don't have that rule. It's a problem for other PHPCS rules that we use, since PHPCS has only single file scope. We'll need to find some solution/workaround (cc @kukulich) and hopefully get rid of inheritDocs later.